spec/lib/investec_open_api/client_spec.rb

```diff
@@ -7,6 +7,9 @@
   let(:client) { InvestecOpenApi::Client.new }
 
   before do
+
+    #WebMock.allow_net_connect!(allow_localhost: true)
```

Comment:
Since stubs are being made in the spec, including the before block here, could we remove this?